pkgs/tools/system/btop/default.nix

```diff
@@ -37,6 +39,11 @@ stdenv.mkDerivation rec {
     ${removeReferencesTo}/bin/remove-references-to -t ${stdenv.cc.cc} $(readlink -f $out/bin/btop)
   '';
 
+  postPhases = lib.optionals rocmSupport [ "postPatchelf" ];
+  postPatchelf = lib.optionalString rocmSupport ''
+    patchelf --add-rpath "${lib.getLib rocmPackages.rocm-smi}/lib" $out/bin/btop
```

Comment:
Technically, $out/bin/btop should be quoted too.

Reply:
Or the other way around, remove all quotes as the quotes for the former (`--add-rpath`) are optional in the case of nix store paths.

Reply:
They're both guaranteed to be non-empty and space-free so do as you will